/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/haliax/nn/__init__.py:159: UserWarning: target_y has dtype uint8, which is not a floating point type. This is probably a mistake.
  warnings.warn(
Step: 0 | Average Train Loss: 200634496.00 | Average Val Loss: 195769936.00
Saving checkpoint to
step: 1, loss: 9960122.0000 time: 5214.09ms, mfu: 0.00%
step: 2, loss: 9461859.0000 time: 2897.31ms, mfu: 0.00%
step: 3, loss: 9555678.0000 time: 246.59ms, mfu: 0.01%
step: 4, loss: 9786683.0000 time: 250.47ms, mfu: 0.01%
step: 5, loss: 10471158.0000 time: 234.86ms, mfu: 0.01%
step: 6, loss: 9928132.0000 time: 239.06ms, mfu: 0.02%
step: 7, loss: 9989962.0000 time: 230.63ms, mfu: 0.02%
step: 8, loss: 10311239.0000 time: 239.27ms, mfu: 0.02%
step: 9, loss: 9657335.0000 time: 238.97ms, mfu: 0.02%
step: 10, loss: 10193244.0000 time: 271.96ms, mfu: 0.03%
step: 11, loss: 10324733.0000 time: 388.95ms, mfu: 0.03%
step: 12, loss: 9890460.0000 time: 453.39ms, mfu: 0.03%
step: 13, loss: 9685747.0000 time: 281.82ms, mfu: 0.03%
step: 14, loss: 10040424.0000 time: 234.65ms, mfu: 0.03%
step: 15, loss: 9608993.0000 time: 232.90ms, mfu: 0.03%
step: 16, loss: 10296306.0000 time: 244.77ms, mfu: 0.03%
step: 17, loss: 9702808.0000 time: 271.06ms, mfu: 0.03%
step: 18, loss: 10303416.0000 time: 275.65ms, mfu: 0.03%
step: 19, loss: 10365251.0000 time: 342.72ms, mfu: 0.03%
step: 20, loss: 10198219.0000 time: 267.79ms, mfu: 0.03%
step: 21, loss: 10464044.0000 time: 246.13ms, mfu: 0.04%
step: 22, loss: 8530032.0000 time: 235.80ms, mfu: 0.04%
step: 23, loss: 9582692.0000 time: 242.66ms, mfu: 0.04%
step: 24, loss: 9614668.0000 time: 260.33ms, mfu: 0.04%
step: 25, loss: 9681482.0000 time: 248.06ms, mfu: 0.04%
step: 26, loss: 9575581.0000 time: 235.87ms, mfu: 0.04%
step: 27, loss: 10213148.0000 time: 235.16ms, mfu: 0.04%
step: 28, loss: 9932398.0000 time: 239.98ms, mfu: 0.04%
step: 29, loss: 10154866.0000 time: 238.34ms, mfu: 0.04%
step: 30, loss: 9921730.0000 time: 244.50ms, mfu: 0.04%
step: 31, loss: 9757540.0000 time: 254.51ms, mfu: 0.04%
step: 32, loss: 9792378.0000 time: 425.54ms, mfu: 0.04%
step: 33, loss: 9951580.0000 time: 299.58ms, mfu: 0.04%
step: 34, loss: 9774599.0000 time: 427.48ms, mfu: 0.04%
step: 35, loss: 10025502.0000 time: 357.27ms, mfu: 0.04%
step: 36, loss: 10324740.0000 time: 339.17ms, mfu: 0.04%
step: 37, loss: 10032614.0000 time: 248.76ms, mfu: 0.04%
step: 38, loss: 9746876.0000 time: 266.77ms, mfu: 0.04%
step: 39, loss: 10285646.0000 time: 409.18ms, mfu: 0.04%
step: 40, loss: 10199644.0000 time: 244.37ms, mfu: 0.04%
step: 41, loss: 10326162.0000 time: 361.54ms, mfu: 0.04%
step: 42, loss: 10055368.0000 time: 273.34ms, mfu: 0.04%
step: 43, loss: 9788110.0000 time: 247.40ms, mfu: 0.04%
step: 44, loss: 9572734.0000 time: 239.53ms, mfu: 0.04%
step: 45, loss: 9989959.0000 time: 245.03ms, mfu: 0.04%
step: 46, loss: 10000632.0000 time: 239.20ms, mfu: 0.04%
step: 47, loss: 10166947.0000 time: 250.44ms, mfu: 0.04%
step: 48, loss: 9874111.0000 time: 251.77ms, mfu: 0.04%
step: 49, loss: 10279242.0000 time: 240.28ms, mfu: 0.04%
step: 50, loss: 10442736.0000 time: 239.63ms, mfu: 0.04%
step: 51, loss: 9736218.0000 time: 237.71ms, mfu: 0.04%
step: 52, loss: 10307687.0000 time: 234.34ms, mfu: 0.04%
step: 53, loss: 9522275.0000 time: 240.88ms, mfu: 0.04%
step: 54, loss: 9749011.0000 time: 240.80ms, mfu: 0.04%
step: 55, loss: 9877652.0000 time: 404.23ms, mfu: 0.04%
step: 56, loss: 10545088.0000 time: 414.19ms, mfu: 0.04%
step: 57, loss: 10056070.0000 time: 280.49ms, mfu: 0.04%
step: 58, loss: 9908936.0000 time: 254.01ms, mfu: 0.04%
Traceback (most recent call last):
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 394, in <module>
    main()
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/draccus/argparsing.py", line 182, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 391, in main
    trainer.train()
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 356, in train
    self.state, loss = self.train_step(self.train_key)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 305, in train_step
    key, opt_state, model, loss = self._train_step(key)
                                  ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/equinox/_jit.py", line 206, in __call__
    return self._call(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/equinox/_module.py", line 875, in __call__
    return self.__func__(self.__self__, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/equinox/_jit.py", line 200, in _call
    out = self._cached(dynamic_donate, dynamic_nodonate, static)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/haliax/core.py", line 129, in tree_unflatten
    @classmethod
KeyboardInterrupt
Traceback (most recent call last):
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 394, in <module>
    main()
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/draccus/argparsing.py", line 182, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 391, in main
    trainer.train()
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 356, in train
    self.state, loss = self.train_step(self.train_key)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/Desktop/ML Projects/GPT_nano/train.py", line 305, in train_step
    key, opt_state, model, loss = self._train_step(key)
                                  ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/equinox/_jit.py", line 206, in __call__
    return self._call(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/equinox/_module.py", line 875, in __call__
    return self.__func__(self.__self__, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/equinox/_jit.py", line 200, in _call
    out = self._cached(dynamic_donate, dynamic_nodonate, static)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rohanmehta/anaconda3/envs/jax/lib/python3.11/site-packages/haliax/core.py", line 129, in tree_unflatten
    @classmethod
KeyboardInterrupt